test/integration/machine-learning.spec.ts

```diff
@@ -205,6 +261,33 @@ describe('admin.machineLearning', () => {
         });
     });
 
+    it('updates the automl model', function () {
+      this.timeout(60000); // Allow up to 60 seconds for this test.
+      return createTemporaryModel({
+        displayName: 'node-integ-test-update-automl'
+      }).then((model) => {
+        const modelOptions: admin.machineLearning.ModelOptions = {
+          tfliteModel: { automlModel: 'this will be replaced below' },
```

Comment:
Initialize the whole thing below when you have the automl information.

Reply:
I won't be able to access it from the .then(updatedModel) function if it's a local variable in the then(automlRef) function...

Reply:
Why not? It's still be in the scope of the `getAutoMLModelReference().then()`. So everything in that block should see it.